### ISSUE TITLE
Mosfet

### ISSUE THREAD

**Issue by Ranghd95** [Proto-App/Proto-Android#226]
doesn't have any real mosfet-like properties

**Comment by Awen2345** [comment]
Hmm there is no mosfer parameter option, I'm a bit disappointed

**Comment by Awen2345** [comment]
Hmm there is no mosfet parameter option, I'm a bit disappointed

**Comment by Proto-App** [comment]
@Ranghd95 
Yes, we know that MOSFET settings are quite poor. We will try to implement at least  voltage threshold.

**Comment by Awen2345** [comment]
@Proto-App how about Length and Width Mosfet parameters?

**Comment by Ranghd95** [comment]
> @Ranghd95 Yes, we know that MOSFET settings are quite poor. We will try to implement at least voltage threshold.

Yep, it's good to have R(DS) on

> @Ranghd95 Yes, we know that MOSFET settings are quite poor. We will try to implement at least voltage threshold.

yes it would be better if the mosfet has Rds on and Voltage threshold
